recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - python
     - numpy
-    - pvl <1.0
+    - pvl
```

Comment:
Should we specify a range here? Is PVL using semantic versioning where we can use a >= 1.0, <= 2.0 or do we need to do something else?

Reply:
I think we need to specify >= 1.0

Reply:
Should this change happen upstream (in our repo) for the meta and the environment and conda-forge will pull that change or should do we need to make the change in both repos. I can't remember if conda-forge updates the recipe if the upstream recipe updates.